pkg/daemon/ceph/client/config.go

```diff
@@ -162,6 +162,12 @@ func mergeDefaultConfigWithRookConfigOverride(clusterdContext *clusterd.Context,
 		return errors.Wrapf(err, "failed to load config data from %q", k8sutil.ConfigOverrideName)
 	}
 
+	// Remove any debug message setting from the config file
```

Comment:
@travisn FYI

Reply:
`debug ms` is the only output that would affect the operator? I wonder if we should disable everything that starts with "debug" just to be safe.

Reply:
Yes `debug ms` is the only option that would cause the CLI to become verbose. I'm not sure if we can easily remove anything that contains `debug` since the `DeleteKey()` method takes an exact argument.
Introspecting the `[]byte` from the ConfigMap seems too error-prone to me.